src/System.Net.WebHeaderCollection/src/System/Net/WebHeaderCollection.cs

```diff
@@ -238,12 +241,14 @@ public override string[] GetValues(int index)
         //     header      - Name of the header.
         // Return Value:
         //     string[] - array of parsed string objects
-        public override string[] GetValues(string header)
+#pragma warning disable CS8610 // Supress warning: Nullability of parameter 'header' doesn't match overridden member
+        public override string[]? GetValues(string header)
+#pragma warning restore
         {
             // First get the information about the header and the values for
             // the header.
-            HeaderInfo info = HeaderInfo[header];
-            string[] values = InnerCollection.GetValues(header);
+            HeaderInfo info = HeaderInfo[header!];
```

Comment:
Nit: can you add a comment like `// HeaderInfo indexer validates and throws for a null header name`?

Reply:
Actually, nevermind, it's fine